### ISSUE TITLE
Consider saving bonus canvas area

### ISSUE THREAD

**Issue by Noitarud** [SimpleMobileTools/Simple-Draw#276]
Rotating lets you view more canvas! But it gets cut off when saving :(
![image_2023_04_26_22_39_08rotated](https://user-images.githubusercontent.com/98794719/234580049-721fffe5-8760-4e2a-a3e7-64cb77f62021.png)

**Comment by Saijin-Naib** [comment]
Agree. My toddler draws on this a lot, and knows how to pan the canvas, so they make really big drawings that I can only save one tiny viewport from.

I'd consider this a bug honestly.
